Sources/Features/Breadcrumb/BacktraceBreadcrumbFileHelper.swift

```diff
@@ -36,78 +32,75 @@ enum BacktraceBreadcrumbFileHelperError: Error {
 
         super.init()
     }
-
+    
     func addBreadcrumb(_ breadcrumb: [String: Any]) -> Bool {
```

Comment:
normally in our Sdks we also limit breadcrumbs based on the number - mostly we care about the number of breadcrumbs rather than the size. I don't think any logic related to it - do we have it in options?

Reply:
I can add a limit, can you elaborate on this one?